qa/tasks/ceph.py

```diff
@@ -607,12 +607,10 @@ def cluster(ctx, config):
 
     devs_to_clean = {}
     remote_to_roles_to_devs = {}
-    remote_to_roles_to_journals = {}
```

Comment:
:+1: these were already a bit obsolete with our uniform-media machines in sepia. especially once ceph-daemon uses ceph-volume, it makes no sense to keep these around.